packages/cli/scaffold/resource.template.ts

```diff
@@ -1,29 +1,50 @@
-import type { PerformFunction, Resource } from 'zapier-platform-core';
```

Comment:
`PerformFunction` has been deprecated and replaced with types like `CreatePerform`, `PollingTriggerPerform`, and `SearchPerform`. These changes bring this file in line with the updates made to `create.template.ts` for example ([seen here](https://github.com/zapier/zapier-platform/blob/1b3eb1f12c284f11694453c47d3ef26af682cfaa/packages/cli/scaffold/create.template.ts)).